paddle/fluid/platform/device_context.h

```diff
@@ -389,14 +389,14 @@ using ShapeBlob = std::unordered_map<std::string, std::shared_ptr<KeyBlob>>;
 using BlobMap = std::unordered_map<int, std::shared_ptr<ShapeBlob>>;
 
 // default mkldnn session id
-constexpr size_t kMKLDNNSessionID_Default = 0;
+constexpr int kMKLDNNSessionID_Default = 0;
```

Comment:
why change to int? size_t is used for aligning with std::hash(this_thread::gettid()), otherwise it may need a static_cast

Reply:
Got it. I will change.